full-service/src/db/txo.rs

```diff
@@ -376,10 +380,14 @@ impl TxoModel for Txo {
             // transaction), we need to update it
             Ok(txo) => {
```

Comment:
This line is overwriting the `TxOut` that is passed in under the variable name `txo`, which is why the encoding is weird. I would recommend changing the name of either one of them as to preserve both of their information sets